src/datajudge/constraints/uniques.py

```diff
@@ -63,6 +74,31 @@ class Uniques(Constraint, abc.ABC):
     (eager or lazy) of the same type as the type of the values of the column (in their
     Python equivalent).
 
+    Furthermore, the `max_relative_violations` parameter can be used to set a tolerance
+    threshold for the proportion of elements in the data that can violate the constraint
+    (default: 0).
+    Setting this argument is currently not supported for `UniquesEquality`.
+
+    For `UniquesSubset`, by default,
+    the number of occurrences affects the computed fraction of violations.
+    To disable this weighting, set `compare_distinct=True`.
+    This argument does not have an effect on the test results for other `Uniques` constraints,
+    or if `max_relative_violations` is 0.
+
+    By default, the assertion messages make use of sets,
+    thus, they may differ from run to run despite the exact same situation being present.
+    To enforce a reproducible output via (e.g.) sorting, set `output_postprocessing_sorter` to a callable
```

Comment:
I think that for now we can simplify and not add an arbitrary posprocessing step and just make the messages reproducible.

Reply:
The main reason why I wanted to make this postprocessing step so generic is that there may be some users with column datatypes which are not (automatically) sortable. If we always force the output_postprocessing_sorter to my suggested function, existing code would break, with no easy way to fix/turn off that feature.

I initially also considered just adding a binary sorted_output flag, but it seemed less flexible:
1. the above mentioned case, requiring intervention to make the output sortable
2. to give flexibility depending w.r.t ASC/DESC and how to order NULL values, which is quite use-case dependent if they are interesting, or should rather be at the end of the output (possibly cut off)